bin/lms-notify-sms.php

```diff
@@ -329,7 +329,7 @@ function create_message($subject, $template) {
 		JOIN (
 			SELECT " . $DB->GroupConcat('contact') . " AS phone, customerid
 			FROM customercontacts
-			WHERE (type & 1) = 1
+			WHERE ((type & 1) = 1 AND (type & 64) <> 64)
```

Comment:
Wszystko co jest w PHP fajnie byłoby załatwiać zamiast liczbami to zdefiniowanymi stałymi.
Byłoby o wiele bardziej czytelnie. Podobnie złożenia bitów np. poprzez CONTACT_EMAIL | CONTACT_NOTIFICATIONS